### PR TITLE
denominator 4.4.1

### PR DESCRIPTION
**Pull request by adriancole**
* Restores IAM credential functionality for CLI
* Eliminates column collisions in CLI output

cc @quinncomendant

### CODE REVIEW

File: Library/Formula/denominator.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 class Denominator < Formula
   # This is a shell script wrapping a java jar; download as single file using nounzip
```
Comment (adamv) [inline Library/Formula/denominator.rb]: This comment can probably be removed now that the URL has the `.jar` extension in it.

File: Library/Formula/denominator.rb [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-  homepage "https://github.com/Netflix/denominator/tree/master/cli"
-  url "http://dl.bintray.com/content/netflixoss/denominator/denominator-cli/release/4.3.3/denominator?direct",
+  homepage "https://github.com/Netflix/denominator/tree/v4.4.0/cli"
+  url "https://bintray.com/artifact/download/netflixoss/maven/com/netflix/denominator/denominator-cli/4.4.0/denominator-cli-4.4.0-fat.jar",
     :using  => :nounzip
```
Comment (adamv) [inline Library/Formula/denominator.rb]: We auto-select `nounzip` for .jar URLs, so I think this `using` can be removed (but please test, thanks.)

File: Library/Formula/denominator.rb [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
-  sha1 "df078fe7e1c3739ea17dfeced936515026890c1e"
+  sha1 "c54d9a3cef009992c6eb3e08f63f56b9ee5e42ae"
 
   def install
+    mv "denominator-cli-4.4.0-fat.jar", "denominator"
```
Comment (adamv) [inline Library/Formula/denominator.rb]: There is a "renaming syntax" for install: `bin.install "denominator-cli-4.4.0-fat.jar" => "denominator"`

**Comment by adamv** [comment]
Thanks for the update.

**Comment by adriancole** [comment]
thanks for the tips! will address them now.

**Comment by adriancole** [comment]
ok updated and tested

**Comment by adriancole** [comment]
oops.. this will become 4.4.1 soon :)

**Comment by adriancole** [comment]
ok hopefully build passes now.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**